scripts/docker/build_test_visit.sh

```diff
@@ -33,14 +33,13 @@ cd build
   -DVISIT_ENABLE_XDB:BOOL=OFF -DVISIT_PARADIS:BOOL=ON \
   -DVISIT_CREATE_XMLTOOLS_GEN_TARGETS:BOOL=OFF \
   -DVISIT_CONFIG_SITE="/home/visit/visit-config.cmake" ../src
-make manuals
```

Comment:
why no more manuals?

Reply:
We don't need a separate step to build the manuals, they are automatically built during the install. Since the purpose is to build and test the install, I didn't see the need to have them available in the build dir as well.